crypto/x509v3/v3_asid.c

```diff
@@ -232,20 +235,23 @@ int X509v3_asid_add_id_or_range(ASIdentifiers *asid,
 /*
  * Extract min and max values from an ASIdOrRange.
  */
-static void extract_min_max(ASIdOrRange *aor,
+static int extract_min_max(ASIdOrRange *aor,
                             ASN1_INTEGER **min, ASN1_INTEGER **max)
 {
-    OPENSSL_assert(aor != NULL && min != NULL && max != NULL);
+    if (!ossl_assert(aor != NULL && min != NULL && max != NULL))
```

Comment:
This is internal not API. Although checking the min/max is kind of stupid. It's less clear with the aor value, so I changed it to this:

    if (!ossl_assert(aor != NULL))